app/src/renderer/features/conversation/conversationSlice.ts

```diff
@@ -18,13 +18,34 @@ const initialState: ConversationState = {
 
 export const fetchConversation = createAsyncThunk(
   "conversation/fetchConversation",
-  async (sourceUuid: string) => {
+  async (sourceUuid: string, { getState }) => {
     const sourceWithItems =
       await window.electronAPI.getSourceWithItems(sourceUuid);
 
+    // Read journalist UUID if auth data is available in state
+    const state = getState() as RootState;
+    const journalistUUID = state?.session?.authData?.journalistUUID;
+
     // Mark all items in this conversation as seen
-    const itemUuids = sourceWithItems.items.map((item) => item.uuid);
-    await window.electronAPI.addPendingItemsSeenBatch(sourceUuid, itemUuids);
+    const itemUuids = sourceWithItems.items
+      .filter((item) => {
+        // Replies are sent by us, we do not need to mark them as seen
+        if (item.data.kind === "reply") {
```

Comment:
I think we can remove this conditional now and just use the seen_by logic for everything (instead of is_read)